### ISSUE TITLE
Filters in Table

### ISSUE THREAD

**Issue by hanan-commits-work** [iTwin/iTwinUI-react#759]
#### Feature
Greetings.
The "dropdown selection" in the filter was available in 2.0 out of the box using showDistinctValueFilters and the developers didn't need to implement this capability from the scratch. we couldn't find this feature in 3.0, so I assume it is removed, but since it is deprecated in 3.0, we have to reinvent the wheel every time just for this basic functionality that was available in the previous version. So we request that iTwin provides such ease in the new update as well.

#### Examples

This is how the 2.x filters worked, 
![image](https://user-images.githubusercontent.com/89520756/182302085-a5de0b00-1dde-42a9-9c85-245ab2afe0e6.png)

And this is the latest iTwinUi 
![image](https://user-images.githubusercontent.com/89520756/182302159-543ae72f-9217-42c6-b50d-de459995917c.png)

**Comment by mayank99** [comment]
Have you seen the custom examples at https://github.com/iTwin/iTwinUI-react/discussions/691?

**Comment by hanan-commits-work** [comment]
Yes, I did something similar in my application. But the problem is, we have to do this repeatedly without much flexibility. We want to make it as generic as possible, but to do that we would need to create our own wrapper, which is not feasible at the moment.
So we ask that Bentley provides the same functionality it was providing prior to the update.

**Comment by veekeys** [comment]
Hello, @hanan-commits-work

If you have already added one in your application, would you mind contributing to iTwinUI?
We have  [Contributing guide](https://github.com/iTwin/iTwinUI-react/blob/main/CONTRIBUTING.md) which can help you start, so you could add generic component and make it available for everyone.

**Comment by hanan-commits-work** [comment]
Hi. The one I made isnt generic just fits the business requirement of our project and it is based on [691](https://github.com/iTwin/iTwinUI-react/discussions/691).
